tests/acceptance/main-test.js

```diff
@@ -0,0 +1,173 @@
+    //Filling 2 to create 2 libraries
+    //Filling 4 to create 4 authors and a random number of books
+    const libNumberForm = Math.floor(Math.random() * 99) + 1;
+    const authorNumberForm = Math.floor(Math.random() * 100) + 1;
+    await visit('admin/seeder');
+    await fillIn('[data-test="libInput"]', libNumberForm);
+    await click('[data-test="libButton"]')
+    await fillIn('[data-test="authorInput"]', authorNumberForm);
+    await click('[data-test="authorButton"]')
+
+    //Asserting that the numbers of libraries and authors created are equal to the numbers filled in the form
+    //Remeber that the number of libraries is three because I created 1 before in the form
+    //Attention, here we are still in seeder page and the numbers I get from the respective number box.
+    const totalNumberOfLibs = libNumberForm + 1 //had to add 1 because one was created in the form
+    assert.equal(find('[data-test="numberLib"]').textContent, totalNumberOfLibs,
+      'Assert total number of libraries in the corresponding number box');
+    assert.equal(find('[data-test="numberAuthor"]').textContent, authorNumberForm,
+      'Assert total number of authors in the corresponding number box');
+
+    //The numbers of book created by the seeder is always random, so here I´m getting this random number of
+    //books from the respective number box to use later in an assertion
+    let numberLib = (this.element.querySelector('[data-test="numberLib"]')).textContent
+    let numberLibInt = parseInt(numberLib)
+    let numberAuthor = (this.element.querySelector('[data-test="numberAuthor"]')).textContent
+    let numberAuthorInt = parseInt(numberAuthor)
+    let numberBooks = (this.element.querySelector('[data-test="numberBook"]')).textContent
+    let numberBooksInt = parseInt(numberBooks)
+
+    //Asserting that there are 3 libraries cards because 1 I created in the form and two created with the 
+    //seeder
+    //Here the assertion works by counting the html elements with their respective data-test attributes,
+    //so, there must be three .card-body that means three libraries
+    await visit('libraries');
+    assert.dom('.card-body').exists({count: numberLibInt}, 'Assert the correct number of library cards');
+
+    //Asserting the number of authors and books created by the seeder in authors page
+    //(Here I use let numberBooksInt)
+    //Here the assertion works by counting the html elements with their respective data-test attributes.
+    //Its the same as was with the libraries, suppose there is 20 authors, there will be 20 data-test="book"
+    //elements
+    await visit('authors');
+    assert.dom('[data-test="author"]').exists({count: numberAuthorInt}, 'Assert the total number of authors in the table');
+    assert.dom('[data-test="book"]').exists({count: numberBooksInt}, 
+      'Assert the total number of books show in table');
+
+    //Invalidating the session
+    await invalidateSession();
```

Comment:
try logging out instead of using helper

Reply:
@tpetrone tried pressing the button, and is the same